js/script.js

```diff
@@ -0,0 +1,41 @@
+const taskInput = document.getElementById("input-task");
+const addButton = document.getElementById("add-task");
+const taskContainer = document.getElementById("task-container");
+
+const reg = /[&<>"'/`]/ig;
+
+let taskList = [];
+
+const addTaskHandler = () => {
+    const taskTitle = taskInput.value.replace(reg, "").trim();
+    
+    if (!taskTitle.length) {
+        alert("Please enter your task");
+        return;
+    }
+
+    const task = createTask(taskTitle);
+
+    taskList.unshift(task);
+
+    const taskElement = createTaskElement(task);
+
+    taskContainer.innerHTML = taskElement.outerHTML.concat(taskContainer.innerHTML);
```

Comment:
Reading the value of [outerHTML](https://developer.mozilla.org/en-US/docs/Web/API/Element/outerHTML) returns a string containing an HTML serialization of the element and its descendants. Setting the value of outerHTML replaces the element and all of its descendants with a new DOM tree constructed by parsing the specified htmlString.

So in future it may create problems like you may lose event listeners.

Reply:
Bhai outerHTML was not causing any issue in other features, the event listeners were working fine. But still changed it using prepend method (adds a child element at the beginning of the element).

Reply:
Then you can keep it. Thanks.